tldw_Server_API/app/api/v1/endpoints/evaluations/evaluations_recipes.py

```
@@ -1,185 +1,251 @@
+
+
+@recipes_router.post(
+    "/recipes/{recipe_id}/validate-dataset",
+    dependencies=[Depends(require_eval_permissions(EVALS_READ))],
+)
 async def validate_recipe_dataset(
     recipe_id: str,
-    request: RecipeDatasetValidationRequest,
+    payload: dict[str, Any],
+    user_ctx: str = Depends(verify_api_key),
     current_user: User = Depends(get_eval_request_user),
```

Ruff (0.15.7)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)